script/Base.s.sol

```diff
@@ -1,5 +1,6 @@
 // SPDX-License-Identifier: GPL-3.0-or-later
 // solhint-disable no-console
+// forgefmt: disable-start
```

Comment:
Instead of putting this at the top of the file, we can put it inside the function.

```solidity
function populateSegmentAndTrancheCountMap() internal {
  // forgefmt: disable-start

  // --- snip ---

  // forgefmt: disable-end
}
```

So it could just disable formatting for the block and not the whole file.

Reply:
agree with Shub

Reply:
> So it could just disable formatting for the block and not the whole file.

unfortunately, i tested this before and it doesn’t work as expected (idk if it’s intended or a bug in forge fmt). adding `// forgefmt: disable-start` will be applied globally to the file. that is, if you set it within a function scope, it won’t be applied to the specified range 

so i decided to put it at the start of the file

Reply:
sad. if this is a bug in Forge please report it in the Foundry repository

Reply:
Addressed in the latest commit: https://github.com/sablier-labs/v2-core/pull/931/commits/2a3b509fe359ca20ee3a0a88cbea066b0bd60d67